client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -204,23 +213,56 @@ class TransferDomainStep extends React.Component {
 		);
 	}
 
+	startPendingInboundTransfer() {
```

Comment:
Should probably match the function signature to how you're calling it here. https://github.com/Automattic/wp-calypso/pull/22116/files#diff-b6f12a988f32deb53b45011dae0978f7R72